release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,22 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 8 November 2023 - Enhancement - CloudGateway 2.12.4, SupportAssistant 1.5.2, FieldControl 2.9.1, CoreGateway 2.13.3, ArtifactDeployer 1.5.2, Orchestrator 1.4.1 - Dependencies updated [ID_37798] [ID_37797] [ID_37784] [ID_37785] [ID_37800] [ID_37805]
+
+Several dependencies have been updated.
```

Comment:
Is any more info possible here, to make it a bit less vague (e.g. which dependencies and why)?

Reply:
It was mainly a dependency on our own shared code, but also for other dependencies (nugets). Our shared code contained some issues in some cases, but it didn't effect behavior in all the DxMs, some were working fine anyway. It's a change we had to do anyway so we did it immediately for all DxMs to have the latest and greatest versions of the dependencies. 

I would leave this generic and not go in debt as it is not needed